community/kernel/src/main/java/org/neo4j/kernel/impl/store/id/IdGeneratorImpl.java

```diff
@@ -372,6 +367,33 @@ public synchronized void close()
         }
     }
 
+    private void closeChannelAndPutThisInstanceIntoClosedState() throws IOException
```

Comment:
Why? Just because it includes the word _and_? I can certainly rename it to something else. I doesn't make sense to have two separate methods for things that go together.